pkg/manifests/sched/sched.go

```diff
@@ -195,6 +205,10 @@ func GetManifests(plat platform.Platform) (Manifests, error) {
 	if err != nil {
 		return mf, err
 	}
+	mf.RBScheduler, err = manifests.RoleBinding(manifests.ComponentSchedulerPlugin, manifests.SubComponentSchedulerPluginScheduler)
```

Comment:
do we need this RoleBinding in both k8s and ocp flows or only in the former?

Reply:
I saw the warning on both components.
In general, the controller is an OCP flow?